broker/broker.go

```diff
@@ -53,16 +46,36 @@ var composeStatus2State = map[string]brokerapi.LastOperationState{
 	"waiting":  brokerapi.InProgress,
 }
 
-func New(compose compose.Client, config *config.Config, catalog *catalog.ComposeCatalog, logger lager.Logger) (*Broker, error) {
-	if compose == nil {
-		return nil, fmt.Errorf("broker: composer should not be nil")
+type Broker struct {
+	Compose        compose.Client
+	Config         *config.Config
+	ComposeCatalog *catalog.ComposeCatalog
+	Logger         lager.Logger
+	AccountID      string
+	ClusterID      string
```

Comment:
We discusses this offline and agreed that it's not ideal but this needs more thinking/refactoring in the future. We'll keep it like that for now.